### PR TITLE
Upgrade opam files to 2.0

### PR DESCRIPTION
**Pull request by ghost**
Hopefully that should unbreak the CI

### CODE REVIEW

File: ocaml-migrate-parsetree-ocamlbuild.opam [hunk 1]
```diff
@@ -15,5 +15,11 @@ depends: [
   "jbuilder" {build & >= "1.0+beta7"}
   "ocaml-migrate-parsetree"
   "ocamlbuild"
+  "ocaml-version" {>= "4.02.0"}
```
Comment (dra27) [inline ocaml-migrate-parsetree-ocamlbuild.opam]:
```suggestion
  "ocaml" {>= "4.02.0"}
```

File: ocaml-migrate-parsetree.opam [hunk 2]
```diff
@@ -17,5 +17,13 @@ depends: [
   "result"
   "ppx_derivers"
   "dune" {build}
+  "ocaml-version" {>= "4.02.0"}
```
Comment (dra27) [inline ocaml-migrate-parsetree.opam]:
```suggestion
  "ocaml" {>= "4.02.0"}
```

File: ocaml-migrate-parsetree-ocamlbuild.opam [hunk 3]
```diff
@@ -15,5 +15,11 @@ depends: [
...
 ]
-available: ocaml-version >= "4.02.0"
+synopsis: "Ocamlbuild plugin for ocaml-migrate-parsetree"
+description: """
+This package provides an ocamlbiuld plugin that can be used to produce
```
Comment (dra27) [inline ocaml-migrate-parsetree-ocamlbuild.opam]:
```suggestion
This package provides an ocamlbuild plugin that can be used to produce
```

File: ocaml-migrate-parsetree-ocamlbuild.opam [hunk 4]
```diff
@@ -15,5 +15,11 @@ depends: [
...
+optimized on-demand staticlly linked ppx drivers, like what Dune does
```
Comment (dra27) [inline ocaml-migrate-parsetree-ocamlbuild.opam]:
```suggestion
optimized on-demand statically linked ppx drivers, as Dune does
```

**Comment by ghost** [comment]
@dra27, thanks for the review!

**Pull request merged**